tmux/paths.go

```diff
@@ -0,0 +1,35 @@
+package tmux
```

Comment:
Please move this to the `dir` package since it isn't referencing tmux in any way and has some overlap with the code already in the `dir` package.